traffic_ops/traffic_ops_golang/deliveryservice/safe.go

```diff
@@ -65,11 +85,20 @@ func UpdateSafe(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, tx, http.StatusBadRequest, fmt.Errorf("decoding: %s", err), nil)
 		return
 	}
-	if ok, err := updateDSSafe(tx, dsID, dsr); err != nil {
+	if version.Major > 3 && version.Minor >= 0 {
+		if dsr.LongDesc1 != nil {
+			api.HandleErr(w, r, tx, http.StatusBadRequest, errors.New("the Long Description 1 field are no longer supported in API 4.0 onwards"), nil)
```

Comment:
this error message should probably use the json field name, i.e. `longDesc1` just to be clear. Also `are` -> `is`